src/tests/common.rs

```diff
@@ -423,6 +423,7 @@ async fn many_messages() -> Result<()> {
 // bootstrap contacts later.
 #[tokio::test]
 async fn connection_attempts_to_bootstrap_contacts_should_succeed() -> Result<()> {
+    utils::init_logging();
```

Comment:
Should this be done for other tests too or should be removed?

Reply:
It is already done in most tests. This particular one was failing so I added it in for debugging. It can be added in when needed.